pkgs/development/tools/misc/segger-ozone/default.nix

```
@@ -0,0 +1,80 @@
+{ stdenv
+, fetchurl
+, fontconfig
+, freetype
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "segger-ozone";
+  version = "3.22a";
+
+  src = fetchurl {
+    url = "https://www.segger.com/downloads/jlink/Ozone_Linux_V322a_x86_64.tgz";
+    sha256 = "0v1r8qvp1w2f3yip9fys004pa0smlmq69p7w77lfvghs1rmg1649";
+  };
+
+  rpath = stdenv.lib.makeLibraryPath [
+    fontconfig
+    freetype
+    xorg.libICE
+    xorg.libSM
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    xorg.libXrender
+  ]
+  + ":${stdenv.cc.cc.lib}/lib64";
+
+  dontUnpack = true;
+
+  installPhase = ''
+    mkdir -p $out
+    tar xf $src -C $out
+    mv $out/Ozone_Linux_V322a_x86_64/* $out
+    mv $out/Lib $out/lib
+    rmdir $out/Ozone_Linux_V322a_x86_64
+
+    mkdir $out/bin
+    ln -s $out/Ozone $out/bin
+  '';
+
+  postFixup = ''
+    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" "$out/Ozone"
+    patchelf --set-rpath ${rpath}:$out/lib "$out/Ozone"
```

```suggestion
    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" "$out/Ozone" \
      --set-rpath ${rpath}:$out/lib "$out/Ozone"
```

This should work. Did not test it.